### PR TITLE
convert objc_glue.c to D

### PR DESCRIPTION
**Pull request by WalterBright**
Another rote conversion.

### CODE REVIEW

**Comment by WalterBright** [comment]
Blocked by https://github.com/dlang/dmd/pull/6240

File: src/posix.mak [hunk 1]
```diff
@@ -224,7 +224,7 @@ ROOT_SRCS = $(addsuffix .d,$(addprefix $(ROOT)/,aav array ctfloat file \
 GLUE_OBJS = iasm.o
 
 ifeq ($(D_OBJC),1)
-	GLUE_OBJS += objc_glue.o
+#	GLUE_OBJS += objc_glue.o
```
Comment (yebblies) [inline src/posix.mak]: Delete

**Comment by WalterBright** [comment]
@yebblies ready to go!

File: src/posix.mak [hunk 2]
```diff
@@ -277,11 +274,11 @@ ROOT_SRC = $(addprefix $(ROOT)/, array.h ctfloat.h file.h filename.h \
 GLUE_SRC = \
 	toir.h irstate.h iasm.c \
 	toelfdebug.d libelf.d scanelf.d libmach.d scanmach.d \
-	tk.c eh.c gluestub.d objc_glue.c
```
Comment (yebblies) [inline src/posix.mak]: Why are you removing `eh.c`?

Comment (WalterBright) [inline src/posix.mak]: Because I forgot to remove it when eh.c was converted to eh.d. GLUE_SRC is just the manifest.

Comment (yebblies) [inline src/posix.mak]: Oh right, I checked my repo for this but it was 5 months behind.

**Comment by yebblies** [comment]
Auto-merge toggled on

**Pull request merged**